src/lv_objx/lv_bar.h

```diff
@@ -111,13 +119,27 @@ void lv_bar_set_value(lv_obj_t * bar, int16_t value, lv_anim_enable_t anim);
  */
 void lv_bar_set_range(lv_obj_t * bar, int16_t min, int16_t max);
 
+/**
+ * Set the type of bar.
+ * @param bar pointer to bar object
+ * @param type bar type
+ */
+void lv_bar_set_type(lv_obj_t * bar, lv_bar_type_t type);
+
 /**
  * Make the bar symmetric to zero. The indicator will grow from zero instead of the minimum
  * position.
  * @param bar pointer to a bar object
  * @param en true: enable disable symmetric behavior; false: disable
+ * @deprecated As of v7.0, you should use `lv_bar_set_type` instead.
  */
-void lv_bar_set_sym(lv_obj_t * bar, bool en);
+static inline void lv_bar_set_sym(lv_obj_t * bar, bool en)
```

Comment:
Wouldn't be better have a `lv_bar_set_type(lv_obj_t * bar, lv_bar_type_t type)` function?

Reply:
It exists:

https://github.com/littlevgl/lvgl/blob/5cd58c6f6f6cdb4e5a9b20b6bdd4acacfc9619c3/src/lv_objx/lv_bar.h#L122-L127

The `set_sym` and `get_sym` functions are there for backwards compatibility (to save people from modifying their code). If you think they're not necessary I can just remove them.

Reply:
>  If you think they're not necessary I can just remove them.

I see. 
If I needed to choose between minor API break and duplication in the API, I'd choose the minor API break.

We can still add `lv_bar/slider_set/get_sym` to [lv_api_map.h](https://github.com/littlevgl/lvgl/blob/dev-7.0/src/lv_api_map.h).

Reply:
I've moved them there.